src/contentscript/connectedAccounts/connected-accounts.ts

```diff
@@ -0,0 +1,161 @@
+import { initBGFunctions } from 'chrome-extension-message-wrapper'
+import makeBlockie from 'ethereum-blockies-base64'
+import { browser } from 'webextension-polyfill-ts'
+import { connectionCondition } from './connected-accounts-assembly'
+import { Account, VerificationRequest } from './types'
+
+interface IRequestVerificationProps {
+  firstAccountId: string
+  firstOriginId: string
+  firstAccountImage: string
+  secondAccountId: string
+  secondOriginId: string
+  secondAccountImage: string
```

Comment:
Maybe is it better to pass accounts via separate structures with similar properties? Seems that it will allow us to reuse account structures and we'll don't have to map them to properties beginning from `first...` and `second...`  in the dapplets code.
```
{
   first: {
      accountId: 'johnsmith'
      originId: 'twitter',
      accountImage: '....'
   },
   second: {
      accountId: 'johnsmith'
      originId: 'twitter',
      accountImage: '....'
   }
}
```